### PR TITLE
feat: e-lab design changes

### PR DESCRIPTION
**Pull request by simon-hng**

### CODE REVIEW

File: components/Timeline.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                </div>
+                <div className="w-px h-full bg-blue-300 dark:bg-gray-600"></div>
+            </div>
+            <div>
+                <h2 className="inline-block px-4 py-2 mb-4 text-xs font-medium text-gray-100 bg-gradient-to-r from-blue-500 to-blue-900 dark:from-blue-400 dark:to-blue-500 rounded-3xl dark:text-gray-100">
+                    {date} <br />
+                    ({duration})
+                </h2>
+                <div className="relative flex-1 mb-10 bg-white border-b-4 border-blue-200 shadow dark:border-gray-700 rounded-3xl dark:bg-gray-900">
+                    <img
```
Comment (simon-hng) [inline components/Timeline.tsx]: Should be next/image for image optimization

File: components/Timeline.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+export default function Timeline() {
+    return (
+        <div className="justify-center max-w-6xl lg:py-8 md:px-6 z-0">
+            <div className="w-full mx-auto lg:max-w-3xl">
+                <Phase
+                    title="Formation and Exploration"
+                    date="2nd October - 15th October 2023"
+                    duration="2 Weeks"
+                    imageSrc="/assets/e-lab/phases/co-founder.jpg"
```
Comment (simon-hng) [inline components/Timeline.tsx]: Image src seems to be broken in the preview deployment

File: components/Timeline.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
+import Image from 'next/image';
+
+function Phase({ title, date, duration, imageSrc, phaseText }) {
+    return (
+        <div className="relative flex justify-between">
+            <div className="flex flex-col items-center w-10 mr-4 md:w-24">
+                <div>
+                    <div className="flex items-center justify-center w-8 h-8 bg-blue-200 rounded-full dark:bg-gray-600">
+                        <div className="w-4 h-4 bg-blue-600 rounded-full dark:bg-blue-400"></div>
```
Comment (simon-hng) [inline components/Timeline.tsx]: We don't use dark/light theming yet

File: components/Timeline.tsx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+                        src={imageSrc}
+                        alt={title}
+                        className="w-full mb-6"
+                    />
+                    <div className="relative z-20 p-6">
+
+                        <p className="mb-2 text-xl font-bold text-gray-600 dark:text-gray-400">
+                            {title}
+                        </p>
+                        <p className="text-gray-700 dark:text-gray-500">
```
Comment (simon-hng) [inline components/Timeline.tsx]: Text-gray has insufficient contrast to background color

**Review by simon-hng — COMMENTED** [review]
Please format with prettier

**Comment by simon-hng** [comment]
You can add framer Motion to add scroll animations.

E.g. Timeline elements fade in on scroll  

https://www.framer.com/motion/scroll-animations/

File: components/Timeline.tsx (hunk 2 above)
Comment (timonschramm) [reply components/Timeline.tsx]: Where do you see that it is broken, for me it looked fine, where could I check?

Comment (simon-hng) [reply components/Timeline.tsx]: https://website-nextjs-git-venture-timon-softdevtumai.vercel.app/e-lab

File: components/Footer.tsx [hunk 5]
```diff
@@ -10,68 +11,72 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
...
-      <div className="container mx-auto grid gap-8 p-16 md:grid-cols-2">
-        <div>
-          <Image
-            src={"/assets/logo_new_white_standard.png"}
-            className="mb-8 flex text-2xl"
-            alt="Logo"
-            width="128"
-            height="32"
-          />
+      {/* <Section className="mx-auto max-w-7xl "> */}
```
Comment (simon-hng) [inline components/Footer.tsx:14]: dead code

File: components/Timeline.tsx [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+            <div className="h-4 w-4 rounded-full bg-blue-600"></div>
+          </div>
+        </div>
+        <div className="h-full w-px bg-blue-300"></div>
+      </div>
+      <div>
+        <h2 className="mb-4 inline-block rounded-3xl bg-gradient-to-r from-blue-500 to-blue-900 px-4 py-2 text-xs font-medium text-gray-100">
+          {date} <br />({duration})
+        </h2>
+        <div className="relative mb-10 flex flex-1 flex-col rounded-3xl border-b-4 border-blue-200 bg-white shadow">
```
Comment (simon-hng) [inline components/Timeline.tsx]: The border at the bottom looks a bit weird to me

File: components/Timeline.tsx [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+            <p className="mb-2 text-xl font-bold text-gray-900">{title}</p>
+            <p className="text-gray-800">{phaseText}</p>
+          </div>
+        </div>
+      </div>
+    </motion.div>
+  );
+}
+
+export default function Timeline() {
```
Comment (simon-hng) [inline components/Timeline.tsx]: We try to keep everything in components/ reusable. In this case we this component should have the Props

```ts
interface TimelineProps {
phases: {
  title: string,
  date: string,
  duration: string,
  image: string,
  text: string,
  }[]
}
```

The data that is used should be on the page

**Comment by Max-vS** [comment]
the navbar is too large on mobile devices and forces horizontal scroll into whitespace

**Pull request merged**